plugins/flytekit-dbt/flytekitplugins/dbt/schema.py

```diff
@@ -246,16 +246,4 @@ def to_args(self) -> List[str]:
 @dataclass_json
 @dataclass
 class DBTFreshnessOutput(BaseDBTOutput):
```

Comment:
should we have the raw logs coming from `dbt source freshness` ?

Reply:
Done 😄